backend/vcdat/Files.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+def isValidFile(start_path, item):
+    if os.path.isfile(os.path.join(start_path, item)) and item[0] != '.' and item.endswith('.nc'):
```

Comment:
Are all climate / model / data files something.nc?

would it be more generic to first check if a file, if so then try to open with CDMS?